lib/fennec/stun/auth.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Fennec.STUN.Auth do
```

Comment:
Have you considered moving it to `Fennec` namespace? The same as in comment in other PR, this doesn't only apply to STUN (I know I named it like that in the first place 😅 ).